test-integration/test-cloning/tests/02_test_monitored_accounts_limit.rs

```diff
@@ -0,0 +1,79 @@
+use integration_test_tools::IntegrationTestContext;
```

Comment:
Nice! Where you able to inspect from the logs that the evictions took place?
The test has no way of checking that afaik unless it was to query the relevant metrics.

Reply:
it was removed from accountdb, that's for sure, test does check for it, and removal from accountsdb can only take place after eviction. I did check metrics manually, don't see much point in adding extra logic to parse metrics from the tests as well, thus complicating it.